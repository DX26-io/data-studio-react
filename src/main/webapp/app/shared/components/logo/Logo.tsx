import React from 'react';

const Logo = () => {
  return (
    <>
      <span className='logo'>Dx26</span>
    </>
  );
};

export default Logo;
