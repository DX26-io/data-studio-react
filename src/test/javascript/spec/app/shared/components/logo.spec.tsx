import React from 'react';
import { mount } from 'enzyme';
import Logo from 'app/shared/components/logo/logo';

describe('Logo tests', () => {
  let mountedWrapper;
  const wrapper = () => {
    if (!mountedWrapper) {
      mountedWrapper = mount(<Logo />);
    }
    return mountedWrapper;
  };

  it('should render component as expected', function() {
    expect(wrapper()).toMatchSnapshot();
  });
});
