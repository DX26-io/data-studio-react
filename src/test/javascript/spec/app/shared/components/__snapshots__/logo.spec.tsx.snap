// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo tests should render component as expected 1`] = `
<Logo>
  <span
    className="logo"
  >
    Dx26
  </span>
</Logo>
`;
